components/layout_2020/flow/inline.rs

```diff
@@ -364,6 +364,11 @@ struct InlineFormattingContextState<'a, 'b> {
     /// The line breaking state for this inline formatting context.
     linebreaker: Option<LineBreakLeafIter>,
 
+    /// Whether or not a soft wrap opportunity is queued. Soft wrap opportunities are
+    /// queued after replaced content and they are processed when the next text content
+    /// is encountered.
+    queued_soft_wrap_opportunity: bool,
```

Comment:
Should this be called `deferred` soft wrap opportunity to be symmetric with `defer_forced_line_break`?

Reply:
Sure, I've renamed this to `have_deferred_soft_wrap_opportunity`.